canvas_modules/common-canvas/__tests__/_utils_/intl-utils.js

```diff
@@ -25,14 +25,11 @@ const defaultLocale = "en-US";
 const locale = defaultLocale;
 
 export function mountWithIntl(node, inOptions) {
```

Comment:
Since we might want to have both enzyme and @testing-library/react we should probably have separate methods or files.  Then they can both live together.

Reply:
Hi Matt I have included separate methods should I also create new test files for the RTL tests (something like palette-flyout-test-rtl.js) ?

Reply:
No, we can update 1 test file at a time so I don't think we need to rename anything.